library/src/shared/services/bank-account/bank-account.service.ts

```diff
@@ -120,7 +131,22 @@ export class BankAccountService implements OnDestroy {
       .createExternalBankAccount(postExternalBankAccount)
       .pipe(
         catchError((err: any) => {
-          let message = 'There was an error creating a bank account';
+          const message = 'There was an error creating a bank account';
+          this.eventService.handleEvent(LEVEL.ERROR, CODE.DATA_ERROR, message);
+          this.errorService.handleError(new Error(message));
+          return of(err);
```

Comment:
We don't pass the actual error to the event or error services?

Reply:
Good catch. We are supposed to.